test/framework/fixtures.go

```diff
@@ -1,12 +1,15 @@
 package framework
 
 import (
-	"errors"
+	"fmt"
```

Comment:
What generates this type of imports? Usually the go imports are grouped in 3 from my experience, the golang core libraries, external libraries and project's internal libraries. This import structure seems to mess up everything.

Reply:
Import order has always been a mess, each IDE is using a different style.